### PR TITLE
Set up new US campaign ticker

### PR DESCRIPTION
**Pull request by i-hardy**
## What are you doing in this PR?

This adds a new and updated ticker component for the 2023 EOY US campaign and adds it to the checkout, behind the existing campaign switch.

[**Trello Card**](https://trello.com/c/XgWlzhwj)

## Why are you doing this?

There is a desire to revive the old campaign tickers used in previous years to track contributions during specific periods. The existing ticker component uses some difficult-to-maintain SCSS styles, is tightly coupled to the data fetching mechanism, is an older style class component, and is not especially screen reader accessible; the new component addresses these issues.

## Accessibility test checklist
 - [x] [Tested with screen reader](https://webaim.org/articles/voiceover/)
 - [x] [Navigable with keyboard](https://www.accessibility-developer-guide.com/knowledge/keyboard-only/browsing-websites/)
 - [x] [Colour contrast passed](https://www.whocanuse.com/)

### CODE REVIEW

File: support-frontend/assets/components/ticker/tickerContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	TickerCopy,
+	TickerCountType,
+	TickerEndType,
+} from './types';
+
+function getInitialTickerValues(
+	tickerCountType: TickerCountType,
+): Promise<TickerConfigData> {
+	return fetchJson<TickerConfigData>(
+		tickerCountType === 'money' ? '/ticker.json' : '/supporters-ticker.json',
```
Comment (GHaberis) [inline support-frontend/assets/components/ticker/tickerContainer.tsx:15]: @i-hardy the data for a ticker can be retrieved from the support site route `/ticker/${tickerId}.json`. The `tickerId` must match the corresponding key from the ticker config from S3. This config is defined as JSON in S3 in the membership-private bucket, under `/{STAGE}/ticker.conf.json`. 

The `tickerId` for the US campaign should be "US". So the route to retrieve the data for the US will be `/ticker/US.json`. This route works on PROD and CODE, and will return the appropriate data for each environment. Each environments data is written to S3  in the contributions-ticker bucket, under `/{STAGE}/{TickerID}.json`. 

The routes above can be used whether the ticker is reporting on a money target OR a supporter target. 

For DEV if working on a ticker with a money target you can use the dummy route `/ticker.json`. If it's a ticker with a supporter target you can use the dummy route `/supporters-ticker.json`, both these routes contain dummy data that can be manipulated during dev.

This repo contains all this info: https://github.com/guardian/contributions-ticker-calculator

File: support-frontend/assets/pages/supporter-plus-landing/formSections/amountAndBenefits.tsx [hunk 2]
```diff
@@ -27,6 +33,14 @@ export function AmountAndBenefits({
 	addBackgroundToBenefitsList?: boolean;
 	isCompactBenefitsList?: boolean;
 }): JSX.Element {
+	const { internationalisation } = useContributionsSelector(
+		(state) => state.common,
+	);
+	const campaignSettings = getCampaignSettings('Us_eoy_2023');
+
+	const showUSCampaignTicker =
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/supporter-plus-landing/formSections/amountAndBenefits.tsx:41]: I've probably missed it - but where's the switch to make sure this isn't enabled yet?

**Review by jamesgorrie — APPROVED** [review]
Really clear PR and code!

File: support-frontend/assets/pages/supporter-plus-landing/formSections/amountAndBenefits.tsx (hunk 2 above)
Comment (i-hardy) [reply support-frontend/assets/pages/supporter-plus-landing/formSections/amountAndBenefits.tsx:41]: It's `window.guardian.enableContributionsCampaign` or `window.guardian.settings.switches.campaignSwitches.enableContributionsCampaign`. It's driven from the RRCP the same as our other switches for things like payment methods.

Currently we only have an on/off switch for 'enable any campaign' as I think it's only ever been used for one thing at a time in the past. Improving this in the future is something George and I were discussing earlier today.

**Pull request merged**